src/api/offers.js

```diff
@@ -16,23 +17,31 @@ const validateMiddleware = (req, res, next) => {
 export default ({offerService}) => {
   const router = new Router()
 
-  router.get('/getAll', (req, res) => {
-    res.json(offerService.getAllOffers())
+  router.get('/getAll', async (req, res) => {
```

Comment:
I'd like a route like `/offers` a lot more

Reply:
this is a route for development only, will be removed when we're approaching production builds